tests/cypress/tests/standalone/readonly_sorting_test.ts

```diff
@@ -0,0 +1,24 @@
+import DashTable from 'cypress/DashTable';
+
+import { AppMode } from 'demo/AppMode';
+
+Object.values([AppMode.ReadOnly]).forEach(mode => {
+    describe(`sort (readonly), mode=${mode}`, () => {
+        beforeEach(() => {
+            cy.visit(`http://localhost:8080?mode=${mode}`);
+            DashTable.toggleScroll(false);
+        });
+
+        it('can sort', () => {
```

Comment:
Stuck in implementation details -- the table optimizes the dropdowns into labels when non-editable. The tests needs to be aware of that as the holder of the text value is not the same.